### PR TITLE
Update DLS.ini

### PR DESCRIPTION
**Pull request by Michael21107**
changed some values to match the default value and did a minor grammar fix

### CODE REVIEW

File: Templates/DLS.ini [hunk 1]
```diff
@@ -11,15 +11,15 @@ DisabledControls=80,86,27
 
 # Extra patch
 ; Enables or disables the extra patch (Default = false)
-ExtraPatch=true
+ExtraPatch=false
 
 # Developer Mode
 ; Enables or disables the developer mode (Default = false)
-DevMode=true
+DevMode=false
```
Comment (TheMaybeast) [inline Templates/DLS.ini]: `DevMove` is currently enabled prior to release in purpose as all that should have access to it are developers.

File: Templates/DLS.ini [hunk 2]
```diff
@@ -11,15 +11,15 @@ DisabledControls=80,86,27
 
 # Extra patch
 ; Enables or disables the extra patch (Default = false)
-ExtraPatch=true
+ExtraPatch=false
```
Comment (TheMaybeast) [inline Templates/DLS.ini]: Considering `AdvancedHookV.dll` will be packaged with DLSv2 on release, maybe we can consider keeping the ExtraPatch enabled by default as all users will have the ability to use it.

**Review by TheMaybeast — APPROVED** [review]

File: Templates/DLS.ini (hunk 2 above)
Comment (Michael21107) [reply Templates/DLS.ini]: what does the extra patch do?

Comment (TheMaybeast) [reply Templates/DLS.ini]: Allows for toggling extras without having to fix the vehicle.

Comment (Michael21107) [reply Templates/DLS.ini]: i would maybe even make it a required thing, as i cant think of a reason not to use it

**Pull request merged**